osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -412,7 +412,7 @@ protected virtual void Populate(TModel model, [CanBeNull] ArchiveReader archive)
         private ArchiveReader getReaderFrom(string path)
         {
             if (ZipUtils.IsZipArchive(path))
-                return new ZipArchiveReader(Files.Storage.GetStream(path), Path.GetFileName(path));
```

Comment:
was there a reason for this change?

Reply:
Because the path is outside the storage directory, so it would throw an exception.

Reply:
Oh, this is still in a non-imported state. Amazing and scary that it worked until now.